README.md

```diff
@@ -0,0 +1,7 @@
+[![CircleCI](https://circleci.com/gh/giantswarm/microstorage.svg?style=shield)](https://circleci.com/gh/giantswarm/microstorage)
+[![Go Report Card](https://goreportcard.com/badge/github.com/giantswarm/microstorage)](https://goreportcard.com/report/github.com/giantswarm/microstorage)
+
+# microstorage
+
+Key-value storage abstraction library.
```

Comment:
I deliberately was terse here. I'm not sure if we want to encourage anyone to use this library anymore?

Reply:
In that case, would it make sense to explicitly say what it is and that it's not really intended for general usage but that the code is shared in case anyone can get value out of it?

Reply:
Sort of, I'll extend that a bit. :+1: